src/main/java/org/moera/node/text/HtmlProcessor.java

```diff
@@ -0,0 +1,46 @@
+package org.moera.node.text;
+
+import org.intellij.lang.annotations.Language;
```

Comment:
Which package this annotation comes from? Why is it needed?

Reply:
I will remove it. It makes sense only for IntelliJ users